### PR TITLE
Init Spring Boot project

### PR DESCRIPTION
**Pull request by MarekSuchanek**
This is a kick off for FAIR Data Station:

- New Spring Boot project with regular stuff that we have in FAIR Data Point and Train Handler (license headers, code style checks, CI, build info via Actuator)
- Mediating communication with configured FDP using Spring Cloud Gateway (prefixed URLs with `/fdp`).

What else should be done here?

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+						<exclude>**/*.yml</exclude>
+						<exclude>**/*.sparql</exclude>
+						<exclude>**/*.properties</exclude>
+						<exclude>**/*.xml</exclude>
+						<exclude>**/*.json</exclude>
+						<exclude>LICENSE</exclude>
+						<exclude>.gitignore</exclude>
+						<exclude>.dockerignore</exclude>
+					</excludes>
+				</configuration>
```
Comment (kburger) [inline pom.xml]: Maybe we can upgrade to the `4.1` way of using the plugin (using `licenseSets`), and move to include vs exclude (less to maintain), something like

```java
<includes>
  <include>pom.xml</include>
  <include>**/*.java</include>
</includes>
```
(any other files that make sense to put an header on?)

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+          java-version: ${{ env.JAVA_VERSION }}
+          cache: 'maven'
+
+      - name: Verify Maven and Java
+        run: |
+          mvn --version
+
+      - name: Run tests
+        run: |
+          mvn --quiet -U -B org.jacoco:jacoco-maven-plugin:prepare-agent test -Dspring.profiles.active=ci
```
Comment (kburger) [inline .github/workflows/build.yml:36]: Do we need to run a ci-local fdp docker image to run the tests?

**Review by kburger — APPROVED** [review]
Minor comments, looks like a good base to start from.

File: .github/workflows/build.yml (hunk 2 above)
Comment (MarekSuchanek) [reply .github/workflows/build.yml:36]: Well, not yet... but if we ever have some integration tests that include also communication with "real" FDP, then we can add it (we will probably need to later specify what FDP version/s are compatible).

File: pom.xml (hunk 1 above)
Comment (MarekSuchanek) [reply pom.xml]: Updated and included also `Dockerfile`... that is what we currently use and I did not found any *best practice* or recommendation to include also something else 🤷🏻‍♂️

**Pull request merged**